LayoutTests/http/wpt/credential-management/setDigitalCredentialsEnabled.https.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Check that Credential Management parts of the API are not exposed by
+            default
+        </title>
+    </head>
+    <body></body>
+    <script>
+        function log(text) {
+            const p = document.createElement("p");
+            p.innerText = text;
+            document.body.append(p);
+        }
+
+        testRunner.dumpAsText();
+
+        if (navigator.credentials.requestIdentity !== undefined){
+            log("FAIL: navigator.credentials.requestIdentity() must not be exposed by default.");
+        }
+
+        if (window.DigitalCredential !== undefined) {
+            log("FAIL: DigitalCredential interface must not be exposed by default.");
+        }
+
+        window.internals.settings.setDigitalCredentialsEnabled(true);
+
+        if (navigator.credentials.requestIdentity){
+            log("FAIL: navigator.credentials.requestIdentity() must be exposed. Was enabled by pref.");
+        }
+
+        if (window.DigitalCredential) {
```

Comment:
The PR says "DigitalCredential should inherit from Credential interface" so I think the test should at least cover that?

Reply:
Yeah, good call. Hopefully the updated test is sufficient.